packages/packages/store/src/__tests__/store.test.tsx

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```
(or import `FC` directly)

Reply:
Done.